roadmap/implementers-guide/src/node/dispute/participation.md

```diff
@@ -69,3 +81,14 @@ Nothing to store
   1. includes `CandidateReceipts`
   1. includes `Vote`s
 1. Delete all relevant data associated with this dispute
+
+### Blacklisting bad block and invalid decendants
+
+In case a block was disputed successfully, and is now deemed invalid.
+
+1. Query `ChainApiMessage::Descendants` for all descendants of the disputed `Hash`.
+1. Blacklist all descendants of the disputed block `ChainApiMessage::Blacklist`.
```

Comment:
Should this not be implicit from `ChainApiMessage::Blacklist`?

Reply:
That would probably be the easier path.

Reply:
It's like pruning a tree. If you cut off one of the branches, you are implicitly removing all the branches and leaves coming off of it.